server/images.go

```diff
@@ -83,9 +84,14 @@ type LayerReader struct {
 }
 
 type ConfigV2 struct {
+	ModelFamily llm.ModelFamily `json:"model_family"`
+	ModelType   llm.ModelType   `json:"model_type"`
```

Comment:
CC @pdevine in case you have feedback on the config structure

Reply:
I think this is good, although I'm wondering if we should make the struct one level deeper to accomodate other non-LLM models in the future (e.g. other foundation models, etc.). Alternatively we keep them at this level (which is probably OK too given any model is going to have a family/type/filetype) but they are instead a format that we specify.

Reply:
This should extend to other GGML models such as Falcon or StarCoder. Beyond that requires refactoring the the GGML interface.

Reply:
My point was if there was a way to future-proof this for non-ggml/llm models in the future.